### PR TITLE
openpose sendWebcam example + socket local IP

### PR DESCRIPTION
**Pull request by orgicus**
I've noticed the openpose example was missing so added one, mostly adapted form the `receivesOnly` and `sendImage` existing examples.

Additionally, a couple of examples were not using `127.0.0.1` (using `10.0.1.5` instead) so I fixed that.

HTH,
George

### CODE REVIEW

File: im2txt/sendWebcam/script.js [hunk 1]
```diff
@@ -30,7 +30,7 @@
 // Create a connection to the Runway HTTP Server
 // You should select HTTP from the INPUT Panel
 // *You should update this address to match the URL provided by the app
-var socket = io.connect('http://10.0.1.5:33100/query');
+var socket = io.connect('http://127.0.0.1:33100/query');
```
Comment (cvalenzuela) [inline im2txt/sendWebcam/script.js:33]: great, thanks for catching this!

File: openpose/sendImage/script.js [hunk 2]
```diff
@@ -64,13 +64,21 @@ document.addEventListener("DOMContentLoaded", function(event) {
...
   // *You should update this address to match the URL provided by the app
-  socket = io.connect('http://10.0.1.5:33200/query');
+  socket = io.connect('http://127.0.0.1:33200/query');
 
   // When a connection is established
   socket.on('connect', function() {
     status.innerHTML = 'Connected';
   });
 
+  socket.on('connect_error', (error) => {
```
Comment (cvalenzuela) [inline openpose/sendImage/script.js]: can you add a small description? this helps newcomers understand socket events.

File: openpose/sendImage/script.js [hunk 3]
```diff
@@ -64,13 +64,21 @@ document.addEventListener("DOMContentLoaded", function(event) {
...
+    console.error(error);
+  });
+
+  socket.on('connect_timeout', (timeout) => {
```
Comment (cvalenzuela) [inline openpose/sendImage/script.js]: same here

File: openpose/sendWebcam/script.js [hunk 4]
```diff
@@ -15,4 +15,130 @@
...
+// ===============================================================
+//
+// Runway: OpenPose Single Image Demo
+// This example sends one image to Runway and draw all humans detected in the image
+// p5.js is used to draw the human data sent from Runway
+// You should select HTTP from the Input Panel
+// 
+// Cristóbal Valenzuela
+// cris@runwayml.com
+//
```
Comment (cvalenzuela) [inline openpose/sendWebcam/script.js]: add your name here!

**Comment by cvalenzuela** [comment]
thanks so much! this looks great!

I added some very small comments.

**Comment by orgicus** [comment]
@cvalenzuela thank you very much for the feedback.
I've added the comments, my name and removed ping-pong handlers (it's just clutter for beginners)

**Comment by cvalenzuela** [comment]
awesome!!

**Pull request merged**